pkg/clusterd/inventory/disk.go

```diff
@@ -101,6 +102,11 @@ func discoverDevices(executor exec.Executor) ([]*LocalDisk, error) {
 	}
 
 	for _, d := range devices {
+		isRBD := regexp.MustCompile("rbd[0-9]+")
```

Comment:
could you add a unit test around this change? While the regex looks good, I've also seen too often when regex causes unexpected side effects. The existing unit tests around this method are pretty light because of the system calls and different results depending on where it's running. The best approach would likely be to factor this change into a method such as `func ignoreDevice(string) bool`, which can be easily unit tested and we can also add more filters to in the future. thanks!